File: Projets/Awesome Restaurant/Awesome Restaurant/Models/Core Data Generated Models/Restaurant+CoreDataProperties.m
//
//  Restaurant+CoreDataProperties.m
//  Awesome Restaurant
//
//  Created by Ludovic Ollagnier on 08/09/2017.
//  Copyright © 2017 Ludovic Ollagnier. All rights reserved.
//
//

#import "Restaurant+CoreDataProperties.h"

@implementation Restaurant (CoreDataProperties)

+ (NSFetchRequest<Restaurant *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Restaurant"];
}

@dynamic name;
@dynamic address;
@dynamic style;
@dynamic alreadyVisited;
@dynamic notation;

@end

File: Projets/Swift project/Swift project/Restaurant+CoreDataProperties.swift
//
//  Restaurant+CoreDataProperties.swift
//  Swift project
//
//  Created by Ludovic Ollagnier on 12/09/2017.
//  Copyright © 2017 Ludovic Ollagnier. All rights reserved.
//
//

import Foundation
import CoreData


extension Restaurant {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Restaurant> {
        return NSFetchRequest<Restaurant>(entityName: "Restaurant")
    }

    @NSManaged public var name: String?

}
